.github/workflows/devcontainer-feature-release.yml

```diff
@@ -0,0 +1,106 @@
+name: Build dev container feature image
+
+env:
+  IMAGE_NAME: ${{ github.repository }}
+  REGISTRY: ghcr.io
+
+on:
```

Comment:
We should probably add a trigger for PRs here so the test jobs run on PRs.

Reply:
Yes, if keeping these in the same workflow yaml file.

Reply:
However, don't use this suggestion since the build/test job is [separated out](.github/workflows/devcontainer-feature-test.yml).